broker-cli/commands/wallet.js

```diff
@@ -170,7 +183,7 @@ module.exports = (program) => {
 
       // TODO: Figure out a way to handle subArguments that could be dynamic
       // for each command
-      let [symbol = ''] = subArguments
+      let [symbol = '', amount] = subArguments
```

Comment:
why does `symbol` default to empty string but `amount` doesn't?

Reply:
@treygriffith we can default it for now